mentorship/gsoc/2022/project-ideas.md

```diff
@@ -83,6 +83,16 @@ Summary:
 - Difficulty: Medium/High
 - Potential mentor(s): Oleg Nenashev, Brad McCoy
 
+<a name="keptn-gitops-integration"></a>
```

Comment:
Hey @thschue, can you add this project to the **2022 Project Ideas** table of content at the start of this document? Thanks!